Implement Value Caching and Diffing for Optimized Change Detection (#7)
bramus/style-observer: README.md

    /* This is called whenever there are changes */
    (values) => {
        console.log(values['--variable1'], values['--variable2'], values['display']);
    },                                                 
    /* Configuration options */
    {

>       notificationMode?: NotificationMode.INDIVIDUAL

Reviewer: ```suggestion
      notificationMode?: NotificationMode.CHANGED_ONLY
```

cssStyleObserver.detach();               /* Detach observer */
```

### Configuration options

> * notificationMode (NotificationMode, default: INDIVIDUAL): Determines whether to observe and return all properties (ALL) or only the changed ones (INDIVIDUAL)

Reviewer: ```suggestion
* `notificationMode` (`NotificationMode`, default: `CHANGED_ONLY`): Determines whether to pass all properties (`ALL`) or only the changed ones (`CHANGED_ONLY`) into the callback
```
